src/app/collective-rewards/hooks/useGetIsWhitelistedBuilder.ts

```diff
@@ -22,3 +23,23 @@ export const useGetIsWhitelistedBuilder = (builder: Address) => {
     error,
   }
 }
+
+export const useGetIsWhitelistedBuilderV2 = (builder: Address) => {
+  const { data, isLoading, error } = useReadContract({
+    abi: BuilderRegistryAbi,
+    address: SponsorsManagerAddress,
+    functionName: 'isBuilderOperational',
+    args: [builder],
+    query: {
+      // 30 seconds, it's the mean time required for a block to be included in RSKJ
+      refetchInterval: 30_000,
```

Comment:
Could we store it in a shared constant value?

Reply:
we could, but I'd rather not to that in this PR as that refactor is going to touch quite a lot of files.

Reply:
done